pages/locationdetail/[id].js

```diff
@@ -13,6 +21,7 @@ export default function LocationDetailsPage({ locations, colonies }) {
     <>
       <LocationCard location={result} />
       <ColonyList colonies={colonies} />
+      <StyledLink href="/">←</StyledLink>
       <StyledAddButton href={"/addcolony"} />
```

Comment:
Kleiner Seitenhinweis:
Falls du doch bei deiner Struktur mit einer Page bleiben möchtest, kannst du vielleicht auch hier href={`/addcolony/${id}`} verwenden? Dann müsstest du allerdings deine pages-Struktur anpassen und eine dynamische Seite bauen ( /addcolony/[is].js ). Auf der hättest du dann jedenfalls die id der location zur Verfügung.

Reply:
Danke für den Hinweis, schaue ich dann bei der Umstrukturierung!